packages/react-native-web/src/modules/injectResponderEventPlugin/index.js

```diff
@@ -39,14 +39,29 @@ ResponderEventPlugin.eventTypes.selectionChangeShouldSetResponder.dependencies =
 ResponderEventPlugin.eventTypes.scrollShouldSetResponder.dependencies = [topScroll];
 ResponderEventPlugin.eventTypes.startShouldSetResponder.dependencies = startDependencies;
 
+let lastActiveTouchTimestamp = null;
+
 const originalExtractEvents = ResponderEventPlugin.extractEvents;
 ResponderEventPlugin.extractEvents = (topLevelType, targetInst, nativeEvent, nativeEventTarget) => {
   const hasActiveTouches = ResponderTouchHistoryStore.touchHistory.numberActiveTouches > 0;
+
+  let shouldSkipMouseAfterTouch = false;
+  if (nativeEvent.type.indexOf('touch') > -1) {
+    lastActiveTouchTimestamp = Date.now();
+  } else if (lastActiveTouchTimestamp && nativeEvent.type.indexOf('mouse') > -1) {
+    const now = Date.now();
+    shouldSkipMouseAfterTouch = lastActiveTouchTimestamp - now < 250;
```

Comment:
This  doesn't work when there is a `window.alert`  call

Reply:
What does that mean?

Reply:
it means if you call window.alert in onPress , and then shouldSkipMouseAfterTouch is not work right

Reply:
Ok. Maybe don't call alert

Reply:
Right, But people maybe can't understand this. Is there a way to record timestamp after onPress called tick